client/src/pages/CreateAction.js

```diff
@@ -1,25 +1,154 @@
 import "./CreateAction.css";
 import Card from "../components/Card";
-import HeaderGoForward from "../components/Card_components/HeaderGoForward";
+import HeaderActionGoForward from "../components/Card_components/HeaderActionGoForward";
 import SpeechInput from "../components/SpeechInput";
-import { useParams } from "react-router";
+import { useParams, useHistory } from "react-router";
+import { useState, useEffect } from "react";
 import FooterSubmit from "../components/Card_components/FooterSubmit";
 
 export default function CreateCase() {
+  /** VARIABLE SET UP */
+  const reminderArray = JSON.parse(localStorage.getItem("newEntry"));
   const { number } = useParams();
+  const indexNumber = Number(number);
+  const history = useHistory();
+  const [verbInput, setVerbInput] = useState(
+    reminderArray.tasks[indexNumber - 1].verb
```

Comment:
I thought `reminderArray` is an array. But in this line it looks like, it is an object? I'm confused. Can you explain?

Reply:
My bad, I renamed it to a more sensible name